### DISCUSSION TITLE
Is there a GrapesJS Discord Community?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5209] [Ideas]
It would be nice to have a centralized place to ask fellow grapers for help, show our creations, and more. Just an idea @artf !

**Comment by artf** [comment]
There is a link in the README

<img width="357" alt="Screenshot 2023-07-04 at 14 20 42" src="https://github.com/GrapesJS/grapesjs/assets/11614725/ee4a81b1-b8a9-44fe-955a-2cc445a6c453">
